ehitchhiking-ui/src/app/app.module.ts

```diff
@@ -7,18 +7,24 @@ import {FormsModule} from '@angular/forms';
 import {MainScreenModule} from './pages/main-screen/main-screen.module';
 import {MenuModule} from './shared/components/menu/menu.module';
 import {RatePassengersModalModule} from './shared/components/rate-passengers-modal/rate-passengers-modal.module';
+import {MenuComponent} from './shared/components/menu/menu.component';
+import { RegistrationComponent } from './shared/components/registration/registration.component';
```

Comment:
RegistrationComponent should have it's personal module

Reply:
Kindly look at RatePassengersModalModule

Reply:
fix